components/org.wso2.carbon.metrics.impl/src/test/java/org/wso2/carbon/metrics/impl/CounterTest.java

```diff
@@ -15,8 +15,7 @@
  */
 package org.wso2.carbon.metrics.impl;
 
-import java.util.Random;
-
+import junit.framework.TestCase;
```

Comment:
All unit tests in Carbon Metrics are written in JUnit. Do you want us to stop using JUnit and use TestNG?